website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+page_title: Enable Vault telemetry
+description: >-
+  Step-by-step guide to enabling telemetry gathering with Vault
+---
+
+# Enable Vault telemetry gathering
+
+Collect telemetry data from your Vault installation.
+
+## Before you start
+
+- **You must have Vault 1.14 or later installed and running**.
+- **You must have access to your [Vault configuration](/vault/docs/configuration) file**.
+
+
+## Step 1: Choose an aggregation agent
+
+@include 'telemetry/supported-aggregation-agents.mdx'
+
+## Step 2: Enable at least one audit device
+
+To include audit-related metrics, you must enable auditing on at least device
+with the `vault audit enable` command. For example, to enable auditing for the
+`file` device and save the logs to `/var/log/vault_audit.log`:
+
+```shell-session
+$ vault audit enable file file_path=/var/log/vault_audit.log
+```
+
+By default, Vault replicates audit device configuration to all nodes within a
```

Comment:
```suggestion
Vault replicates audit device configuration to all nodes within a
```

I don't think `local` changes how auditing works within a single cluster.

Reply:
One of the existing docs made a point of including the `-local` flag to avoid replication and implied that it applies to replication within a cluster and DR-related replication. Is it strictly DR replication that it affects?

Reply:
My understanding is:
* within a cluster, both local and non-local audit devices are used by all nodes in the cluster when servicing requests
* across DR clusters, all local and non-local audit devices are replicated to the DR secondary
* across PR clusters, only non-local audit devices are replicated to the PR secondary (which can also have its own independent local audit devices)

Reply:
I tweaked the language to focus on why you might want to use the local flag. Let me know if it makes more sense.